src/index.html

```diff
@@ -1,15 +1,181 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form class="form1" action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      onsubmit="onSubmit()"
+      autocomplete="off"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <div class="line">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            required
+          >
+        </div>
+
+        <div class="line">
+          <label for="name">Name:</label>
+          <input type="text"
+            name="name"
+            id="name"
+            required
+          >
+        </div>
+
+        <div class="line">
+          <label for="age">How old are You?</label>
+          <input type="number"
+            name="age"
+            id="age"
+            min="1"
+            max="150"
+            value="12"
+          >
+        </div>
+
+        <div class="line">
+          <label for="birthday">Full date of birth:</label>
+          <input type="date"
+            name="birthday"
+            id="birthday"
+          >
```

Comment:
переноси все атрибуты, первый тоже
```suggestion
          <input
            type="date"
            name="birthday"
            id="birthday"
          >
```